### PR TITLE
fix large scroll-id using post-string instead of get

### PR DESCRIPTION
**Pull request by nikopol**
hello,
here is a fix proposal for issue #150 based on tag v2.1.0.

I just give the scroll-id in the body, as documented here http://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-scroll.html

### CODE REVIEW

**Comment by michaelklishin** [comment]
This has conflicts with master. Please rebase.

File: project.clj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(defproject clojurewerkz/elastisch "2.1.0"
+(defproject clojurewerkz/elastisch "2.1.1-SNAPSHOT"
```
Comment (michaelklishin) [inline project.clj]: Never update versions in a pull request. It's up to the maintain to decide when to do it and how.

Comment (michaelklishin) [inline project.clj]: There is a separate branch, `2.1.x-stable`, for `2.1.x` releases.

**Comment by nikopol** [comment]
i've re-done it from branch 2.1.x-stable

**Pull request closed without merging**